lte/gateway/c/core/oai/tasks/sgw/sgw_paging.c

```diff
@@ -31,19 +31,33 @@
 #include "lte/gateway/c/core/oai/lib/itti/itti_types.h"
 #include "lte/gateway/c/core/oai/include/s11_messages_types.h"
 
-void sgw_send_paging_request(const struct in_addr* dest_ip) {
-  OAILOG_DEBUG(
-      TASK_SPGW_APP, "Paging procedure initiated for ue_ipv4: %x\n",
-      dest_ip->s_addr);
+void sgw_send_paging_request(
+    const struct in_addr* dest_ipv4, const struct in6_addr* dest_ipv6) {
   MessageDef* message_p                       = NULL;
   itti_s11_paging_request_t* paging_request_p = NULL;
 
   message_p =
       DEPRECATEDitti_alloc_new_message_fatal(TASK_SPGW_APP, S11_PAGING_REQUEST);
   paging_request_p = &message_p->ittiMsg.s11_paging_request;
   memset((void*) paging_request_p, 0, sizeof(itti_s11_paging_request_t));
-  paging_request_p->ipv4_addr = *dest_ip;
 
+  if (dest_ipv6) {
+    char ip6_str[INET6_ADDRSTRLEN];
+    inet_ntop(AF_INET6, dest_ipv6, ip6_str, INET6_ADDRSTRLEN);
+    OAILOG_DEBUG(
+        TASK_SPGW_APP, "Paging procedure initiated for ue_ipv6: %s\n", ip6_str);
```

Comment:
not related to your changes but for OAILOG_DEBUG calls the SPGW value should be `LOG_SPGW_APP`, can you please fix those usages?

Reply:
I will take this up as a separate PR